controllers/policyserver_controller.go

```diff
@@ -172,7 +171,7 @@ func (r *PolicyServerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				},
 			}
 		})).
-		Watches(&source.Kind{Type: &policiesv1.ClusterAdmissionPolicy{}}, handler.EnqueueRequestsFromMapFunc(func(object client.Object) []reconcile.Request {
+		Watches(&policiesv1.ClusterAdmissionPolicy{}, handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, object client.Object) []reconcile.Request {
```

Comment:
I have been trying to understand from where these contexts come now, as they now seem to come from the Controller. Particularly, I am wondering on what happens if they timeout, as we could stop processing request through the webhook correctly and lock the cluster.

I have only come by this: https://github.com/kubernetes-sigs/controller-runtime/pull/1334.

Reply:
Just in case, on the daily we agreed that this question should not block the merge.